yt/frontends/ramses/tests/test_outputs.py

```diff
@@ -204,3 +204,34 @@ def test_ramses_sink():
 
     for field in expected_fields:
         assert(('sink', 'field') not in ds.field_list)
+
+
+ramses_new_format = "ramses_new_format/output_00011/info_00011.txt"
```

Comment:
I've staged this data file on the test box. The next run of the tests should include this test.

Reply:
Great thanks!